cytoxnet/dataprep/dataprep.py

```diff
@@ -1,2 +1,156 @@
 # potentially useful: deepchem dataset class or pandas split
 # having our data in the deepchem.dataset form at the end is very helpful
+
+
+"""
+Data splitting module for creating train/test splits  
+"""
+
+import pandas as pd
+import deepchem
+import sklearn
+
+
+def convert_cats(dataframe):
+    """
+    Drops rows with missing values and converts categorical values to integers.
+    
+    Parameters
+    ----------
+    dataframe: featurized dataframe
+    
+    Returns
+    -------
+    modified dataframe without any na values in the 'Mol' column 
+    and all categorical columns with string values converted to integer values
+    """
+    # Drop all na values in the 'Mol' column
+    # May want to expand the range of columns beyond 'Mol'
+    
+    dataframe = dataframe.dropna()
+
+    # convert categorical values, if any exist, into integers
+    # might be able to use sklearn Label Encoder or OnHotEncoder
+    # might be necessary if we categorize target areas/tox methods/etc.
+    # Iterate through columns and convert any containing strings to ints
+    for column in dataframe:
+    	column_series_obj = dataframe[column]
+    	if dataframe[column].dtype not numeric:
```

Comment:
non uniform indentation, do 4 spaces or a tab for everything

Reply:
Thanks for pointing that out!  I've been editing on sublime text and didn't realize my indentation was default set to 2 spaces rather than 4.